src/routes/console/project-[project]/auth/security/+page.svelte

```diff
@@ -198,4 +221,28 @@
             </Button>
         </svelte:fragment>
     </CardGrid>
+
+    <CardGrid>
+        <Heading tag="h2" size="6">Session Limit</Heading>
+        <p>
+            Maximum number of active sessions allowed per user per project.
+        </p>
+        <svelte:fragment slot="aside">
+            <form class="form u-grid u-gap-16">
+                <ul class="form-list">
+                    <InputNumber id="max-session" label="Limit" bind:value={maxSessions} />
+                </ul>
+            </form>
+        </svelte:fragment>
+
+        <svelte:fragment slot="actions">
+            <Button
+                disabled={maxSessions === $project.authMaxSessions}
+                on:click={() => {
+                    updateSessionsLimit();
+                }}>
+                Update
+            </Button>
+        </svelte:fragment>
+    </CardGrid>
```

Comment:
Let's wrap this in a form and handle the function on submit, this way we can validate the input (for this to work the button needs to have the submit attribute). Something like this:
```js
<Form on:submit={updateSessionLimit}>
[...]
      <Button
                disabled={maxSessions === $project.authMaxSessions}
                submit>
                Update
            </Button>
[...]
</Form>
```

Reply:
I copied the existing pattern from above section?